.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+# A workflow that pushes artifacts to Sonatype
+name: Release build
+
+on:
+  push:
+    tags:
+      - '*'
+  repository_dispatch:
+    types: [release]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - name: Checkout android-maps-ktx
+      uses: actions/checkout@v2
+
+    - name: Release build
+      run: |
+        echo "Decrypting secret key ring file."
+        gpg --quiet --batch --yes --decrypt --passphrase=${{ secrets.GPG_SECRING_DECRYPTION_PASSPHASE }} 
```

Comment:
@arriolac 

if you use a new enough gradle, can pass in the armor file instead. https://github.com/googlemaps/java-routes/pull/9/files#diff-d2b8236fec89c2b24e0bbec19cc1646aR78

Reply:
Thanks. I'll take a look

Reply:
I can't use `useInMemoryPgpKeys` since the Gradle supported by android build tools is 5.4.1 :/